src/Simplex/Chat/View.hs

```diff
@@ -691,6 +692,18 @@ viewUserProfileUpdated Profile {displayName = n, fullName, image} Profile {displ
   where
     notified = " (your contacts are notified)"
 
+viewContactPrefsUpdated :: Contact -> [StyledString]
+viewContactPrefsUpdated Contact {profile = LocalProfile {preferences}, userPreferences = ChatPreferences {voice = userVoice}} =
+  let contactVoice = preferences >>= voice
+   in ["preferences were updated: " <> "contact's voice is " <> viewPreference contactVoice <> ", user's voice is " <> viewPreference userVoice]
```

Comment:
maybe a different phrasing would be better.. "voice" -> "voice messages", also have separate command for viewing current values? then this would be just "preferences were updated, use <cmd @contact_name> to view current preferences"

Reply:
Not sure about that. Currently it's short and doesn't require any additional commands to execute for viewing a result. However voice messages looks better than voice. Will make it like this